packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -517,3 +517,33 @@ function Demo(props) {
     </ListView>
   );
 }
+
+const manyItems = [];
+for (let i = 0; i < 500; i++) {manyItems.push({key: i, name: `item ${i}`});}
+
+function DisplayNoneComponent(args) {
+  const [isDisplay, setIsDisplay] = useState(false);
+  useEffect(() => {
+    const timeout = setTimeout(() => setIsDisplay(true), 10000);
```

Comment:
Is this weird for the story? Will people know they need to wait 10 seconds? Should we add a button or something instead?

Reply:
+1. Alternatively does this reproduce in a test? Maybe we could write a test for the crash (if it does crash in test without the changes), write a test verifying that the virtualizer isn't rendering any items within itself (maybe not that great of a test since it isn't directly checking for a crash/infinite loop), or make this setup a chromatic test instead so we don't have to remember to manually test this